src/Style.js

```diff
@@ -6,6 +6,18 @@
  * information about them.
  */
 
+var sigmas = require("./fontMetrics.js").sigmas;
+
+var metrics = [];
+
+for (var i = 0; i < 3; i++) {
+    metrics[i] = Object.keys(sigmas).reduce(function(res, key) {
```

Comment:
Eek.  I've been spoiled developing for IE10+.  Not having `Object.keys` is really sad.  It would be nice to have a polyfill.  In order to keep KaTeX small I'd probably want to have a legacy and a modern build.